xml/operations-maintenance-controller-full_recovery_test.xml

```diff
@@ -652,119 +754,24 @@ Start request sent for job 64715c6ce8ed40e1b346136083923260</screen>
 8.0K    /tmp/mysql_restore/winchester
 4.0K    /tmp/mysql_restore/xtrabackup_galera_info</screen>
        </listitem>
+       <listitem>
+         <para>Repeat steps 2-5 on other two controllers where the 
```

Comment:
Did you want to do an example of using ansible to fine the FND-MDB nodes here too?

Reply:
this is missing a "the" .... "Repeat steps 2-5 on other two controllers where the"  should have a "the" between "on" and "other".... "Repeat steps 2-5 on the other two controllers where the"